benches/assets/cloud_render_reconstruct.glsl

```diff
@@ -0,0 +1,79 @@
+#version 460
+#extension GL_GOOGLE_include_directive : enable
+#extension GL_EXT_samplerless_texture_functions : enable
+
+#include "cloud_render_common.glsl"
```

Comment:
Where do these bench files come from? I think glslpp-rs will very quickly produce an error when it encounters the `#include` so we're just benchmarking the parsing of the first few lines.

Also gathering files like this with no license disclosure could be a problem.